templates/main.template

```diff
@@ -52,45 +53,70 @@ Metadata:
       QSS3KeyPrefix:
         default: Quick Start S3 Key Prefix
 Parameters:
-  pDBPassword:
-    Description: Mixed alphanumeric and must be between 8 and 28 characters and contain
-      at least one capital letter
-    NoEcho: true
-    Type: String
-    MinLength: 8
-    MaxLength: 28
-    AllowedPattern: '[a-zA-Z0-9!^*\-_+]*'
-    ConstraintDescription: Can only contain alphanumeric characters or the following
-      special characters !^*-_+, between 8 and 28 characters
-  pNotifyEmail:
-    Description: Notification email address for security events (you will receive
-      a confirmation email)
-    Type: String
-    Default: distlist@example.org
   pEC2KeyPairBastion:
     Description: The SSH key pair in your account to use for the bastion host login
     Type: AWS::EC2::KeyPair::KeyName
   pEC2KeyPair:
     Description: The SSH key pair in your account to use for all other EC2 instance
       logins
     Type: AWS::EC2::KeyPair::KeyName
-  pSupportsConfig:
-    Description: Is AWS Config Rules already configured for this region? Use ''false''
-      if you are uncertain. See AWS Config Management Console or Deployment Guide
-      for details.
-    AllowedValues:
-    - 'true'
-    - 'false'
-    Default: 'false'
-    Type: String
   pAvailabilityZoneA:
     Description: Availability Zone 1
     Type: AWS::EC2::AvailabilityZone::Name
   pAvailabilityZoneB:
-    Description: Availability Zone 2
+    Description: Availability Zone 2. NOTE - This should be different than the AZ above.
```

Comment:
Please deprecate this pattern in favor of a `List<AWS::EC2::AvailabilityZone::Name>`. Here's an example.

https://github.com/aws-quickstart/quickstart-boomi-atom/blob/182fe44a5e3b5a2b35098265558cf229803eea27/templates/boomi-atom-master.template.yaml#L125-L129

Reply:
The link you provided didnt work but I think I figured it out:
  pAvailabilityZoneA:
    Description: Availability Zone 1
    Type: List<AWS::EC2::AvailabilityZone::Name>
  pAvailabilityZoneB:
    Description: Availability Zone 2. NOTE - This should be different than the AZ above.
    Type: List<AWS::EC2::AvailabilityZone::Name>
  MaxPasswordAge:

Reply:
This also breaks the parameter reference to the VPC Prod template....it looks for a string